### ISSUE TITLE
Change "Job title" to "Main occupation" or add explicit hint

### ISSUE THREAD

**Issue by tuttiq** [WWCodeTokyo/speak-her-db#192]
**Background**

On the current form we have "Job title" that can be misinterpreted as any/all professional titles that someone holds (E.g. people being external directors in X company boards, etc). To keep this field simple and clean, we should make it more clear that it is only meant for the "Main occupation" and that additional titles should go on the Speaker Bio field.

<img width="932" alt="Screen Shot 2020-10-12 at 7 28 53 PM" src="https://user-images.githubusercontent.com/1096046/95736439-36a5f780-0cc1-11eb-9d87-715081b161ed.png">

**Suggested changes**

- [ ] Change the field label to "Main occupation"
- [ ] Add a persistent hint `Please add additional titles on the Speaker Bio below` (should we do this? See my comment below -Ann)

**Comment by tuttiq** [comment]
@ann-kilzer @yanarchy Should we also change `Company` and `Organization` fields to match/become more clear? A few options I thought of that could be more clear:
- `Main organization` and `Secondary organization`
- `Organization` and `Secondary organization`
- `Organization` and `2nd Organization`

**Comment by tuul-wq** [comment]
@tuttiq I'd like to work on this issue

**Comment by ann-kilzer** [comment]
The suggested persistent hint "Please add additional titles on the Speaker Bio below" is inconsistent with the Secondary hint "Academic institutions, volunteer positions, or second jobs". We're telling them to add it below, and then immediately asking for them to put it in this field. 

I'd recommend leaving Job Title field as is, and adding the persistent hint "Primary occupation".

@tuul-wq Fine to work on this but let me clarify with @tuttiq first. For translations, if the coder isn't a Japanese speaker, we typically make a follow-up ticket to address those in a later PR.

**Comment by arisaueno** [comment]
@tuttiq @ann-kilzer 

As comments from my end, usually people belong to companies or organizations, but in my case, I'm a freelancer and don't belong to a specific company at all since work is project-based. So I'd like to leave the job title section. 

Other comments
---
Job Title: 
Company/ Organization: 
Secondary Role:  >> I'd suggest this to change to "other activities" so that this section includes more options for various types of people.
Academic institutions, volunteer positions, or second jobs: 
City: >> Maybe we don't need this. It's rare to ask the city name, especially when we already have a prefecture section.
Prefecture:
Speaker Bio: 
A brief description of the nominee written in the third person. (She/ They, etc)

Suggestions and translations for them.
---
Job Title: 肩書き
Company/ Organization: 所属 (means "belong")
Other activities: その他の活動
Academic institutions, volunteer positions, or second jobs: 教育機関、ボランティア活動、複業などの所属があれば
Location (Prefecture): 居住地（都道府県）
Speaker Bio: 簡単なプロフィール
A brief description of the nominee written in the third person. (She/ They, etc): 推薦する人についての概要をご記載ください


Please let me know if there are any questions or what you think.

**Comment by shokokb** [comment]
These are my suggestions. 
Job Title: **役職**(means "role") or 肩書き
Secondary Role:**複数の役職がある場合の役職**
Company/ Organization: 所属 (means "belong")
Other activities: その他の活動 or **仕事外の活動**
Academic institutions, volunteer positions, or second jobs:
教育機関、ボランティア活動、複業などの他の所属がる**場合にご記入ください**
Location (Prefecture): **活動拠点**（都道府県）
Speaker Bio: スピーカーの簡単な経歴
A brief description of the nominee written in the third person. (She/ They, etc):
第三者としてスピーカー（候補者）を推薦できる方がいる場合には推薦文を追加してください

I'll edit more explanations about this comment.

**Comment by ann-kilzer** [comment]
This ticket is focused on the job titles section of the form only, so let's focus on that.

For more general design feedback, please use the slack chat, or open a new ticket.

For translations, let's put all the work into #200 and finish that PR before starting new translations of the same content.

Thanks for your understanding 🙇🏻‍♀️
